lms/static/js/views/message_banner.js

```diff
@@ -37,4 +37,4 @@
 
         return MessageBannerView;
     });
-}).call(this, define || RequireJS.define);
+}).call(this, define || RequireJS.define);
```

Comment:
We did some changes in message_banner which i'm not able to see in this file. You can see them here https://github.com/edx/edx-platform/pull/8082/files#diff-db57c2f627cd8ff4221f70cb6dbac64bR9